app/views/users/show.html.erb

```diff
@@ -1,9 +1,36 @@
-<p id="notice"><%= notice %></p>
+<%= render  "shared/notice" %>
 
-<p>
-  <strong>Username:</strong>
-  <%= @user.username %>
-</p>
-
-<%= link_to 'Edit', edit_user_path(@user) %> |
-<%= link_to 'Back', users_path %>
+<div class="mt-5 pt-4 row mx-0 px-0 justify-content-center m-auto w-100 container">
+    <div class="upcoming eventz row d-flex flex-column m-auto col-12 w-100">
+        <h2 class="mb-5 text-center">Created events</h2>
+        <div class="row d-flex container justify-content-around">
+            <% @user.events.where(user_id: @user.id).each do |event| %>
```

Comment:
Such a call must not be done in the view but in the controllers.
Create in the controller a view variable that will keep that result in order to be used in the views

Also, `@user.events.where(user_id: @user.id)` can simply be written `@user.events`